app/src/main/java/mozilla/lockbox/store/DataStore.kt

```diff
@@ -66,13 +66,8 @@ open class DataStore(
             }
         }.addTo(compositeDisposable)
 
-        val resetObservable = dispatcher.register
```

Comment:
was there a reason this was removed? would be good for users to be able to reset the datastore I feel... this is how we reset from the `DisconnectAccount` button